infrastructure/api-mgmt.tf

```diff
@@ -46,3 +46,27 @@ module "api_mgmt_policy" {
   api_name               = "${module.api_mgmt.name}"
   api_policy_xml_content = "${replace(file("api-policy.xml"), "ALLOWED_CERTIFICATE_THUMBPRINTS", local.quoted_thumbprints)}"
 }
+
+module "api_mgmt_reconciliation" {
+  source        = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+  name          = "${var.component}-reconciliation-api"
+  api_mgmt_name = "core-api-mgmt-${var.env}"
+  api_mgmt_rg   = "core-infra-${var.env}"
+  revision      = "1"
+  product_id    = "${module.api_mgmt_product.product_id}"
+  display_name  = "Blob Router Reconciliation API"
+  path          = "reform-scan/reconciliation-report"
```

Comment:
i think this will mean that the full endpoint for api request will be `{core-api-url}/reform-scan/reconciliation-report/reform-scan-reconciliation-report/{date}`

perhaps just `path = "reconciliation"` is enough?

Reply:
p.s. mind you, all endpoints will be available what's present in current swagger spec at the time of infra execution